src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerListener.cs

```diff
@@ -7,14 +7,15 @@
 using System.Threading.Tasks;
 using Microsoft.Azure.Documents;
 using Microsoft.Azure.Documents.ChangeFeedProcessor;
-using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.Documents.ChangeFeedProcessor.Monitoring;
+using Microsoft.Azure.Documents.ChangeFeedProcessor.PartitionManagement;
 using Microsoft.Azure.WebJobs.Host.Executors;
 using Microsoft.Azure.WebJobs.Host.Listeners;
 using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Azure.WebJobs.Extensions.CosmosDB
 {
-    internal class CosmosDBTriggerListener : IListener, IChangeFeedObserverFactory
+    internal class CosmosDBTriggerListener : IListener, Microsoft.Azure.Documents.ChangeFeedProcessor.FeedProcessing.IChangeFeedObserverFactory
```

Comment:
nit: Can you add a `using` for this?

Reply:
Sadly I cannot. Since we support the old way of creating the Observers and Processors (marked as obsolete) in a different namespace but with the same class names, Im forced to add these namespaces here to tell the code I want the new version.